### PR TITLE
cli: warn if Chrome died on its own instead of exit

### PR DESCRIPTION
**Pull request by patrickhulce**
**Summary**
Trying this out to see if it fixes the Windows flake. I think it's a good idea either way honestly.

### CODE REVIEW

File: lighthouse-cli/test/smokehouse/test-definitions/redirects/redirects-config.js [hunk 1]
```diff
@@ -18,6 +18,6 @@ module.exports = {
       'redirects',
     ],
     // Use provided throttling method to test usage of correct navStart.
-    throttlingMethod: 'provided',
+    throttlingMethod: /** @type {'provided'} */ ('provided'),
```
Comment (patrickhulce) [inline lighthouse-cli/test/smokehouse/test-definitions/redirects/redirects-config.js:21]: I have no earthly idea why my other changes would start triggering a failure here, but [they did](https://github.com/GoogleChrome/lighthouse/runs/895613104#step:11:6) might as well fix it

**Comment by patrickhulce** [comment]
It passed the first go around, but I'll keep retrying it to make sure it's doing its job.

File: lighthouse-cli/run.js [hunk 2]
```diff
@@ -8,6 +8,7 @@
 /* eslint-disable no-console */
 
 const path = require('path');
+const psList = require('ps-list');
```
Comment (patrickhulce) [inline lighthouse-cli/run.js:11]: doing this in a crossplatform way is a gigantic pain, a CLI-only dep that's pretty small seems ok for this

File: lighthouse-cli/test/smokehouse/test-definitions/redirects/redirects-config.js (hunk 1 above)
Comment (connorjclark) [reply lighthouse-cli/test/smokehouse/test-definitions/redirects/redirects-config.js:21]: weird

**Pull request merged**